_updates/2023-05-16.html

```diff
@@ -1,18 +1,17 @@
 ---
 ---
 <div class="ds-u-padding-y--2">
-  <div class="ds-u-font-size--h3 ds-u-font-weight--bold">
-    Version 1 of the BCDA Partially Adjudicated Claims Enhancement is now live for production use.
-  </div>
-  <div class="ds-u-font-size--lead ds-u-padding-y--1">
-    May 16, 2023
-  </div>
   <div>
     <p>
-        The Centers for Medicare & Medicaid Services (CMS) is pleased to announce the initial production release of the Beneficiary Claims Data Application Programming Interface (BCDA) Partially Adjudicated Enhancement on May 16th, 2023. This initial release was designed as an expansion of the data available to Medicare providers from BCDA and is available to Accountable Care Organizations (ACOs) participating in the ACO Realizing Equity, Access, and Community Health (ACO REACH) Model. Access to partially adjudicated claims data reduces the time to access Parts A and B fee-for-service claims to 2-4 days after claim submission, while it is typically up to 14 days for adjudicated claims data. As ‘partially adjudicated’ claims, these claims have entered the Medicare processing system, but are not paid or fully processed by CMS and may be subject to changes. The data is delivered through the Claim and ClaimResponse bulk Fast Health Interoperability (FHIR®) specification resource types. Part D (drug coverage) and Durable Medical Equipment (DME) are not included in partially adjudicated claims. This enhancement can be used to increase care coordination, improve healthcare outcomes, and reduce overall healthcare cost for Medicare beneficiaries.
-    </p>
-    <p>
-        For more information on partially adjudicated claims, the User Guide can be found <a href="/assets/data/BCDA_Partially_Adjudicated_Claims_Enhancement_User_Guide_v1.0.0.pdf" target="blank" class="in-text__link">here</a>.
+      The Centers for Medicare & Medicaid Services (CMS) is pleased to announce the initial production release of Beneficiary
+      Claims Data Application Programming Interface (BCDA) Partially Adjudicated Enhancement on May 16th, 2023. This initial
+      release was designed as an expansion of the data available to Medicare providers from the BCDA and is available to
+      Accountable Care Organizations (ACOs) participating in the ACO Realizing Equity, Access, and Community Health (ACO
+      REACH) Model.
     </p>
+    <p><strong>
+        For more information on partially adjudicated claims, the User Guide can be found <a href="/assets/data/BCDA_Partially_Adjudicated_Claims_Enhancement_User_Guide_v1.0.0.pdf" target="blank" class="in-text__link">here</a> and the Data Dictionary can be found can be found <a href="https://confluence.cms.gov/assets/data/BCDA_Partially_Adjudicated_Claims_Enhancement_User_Guide_v1.0.0.pdf" target="blank"
+          class="in-text__link">here</a>.
+    </strong></p>
   </div>
 </div>
```

Comment:
Oh this is part of the other ticket for the updates page, right? Can we remove this from the homepage updates? :D Thanks!

Reply:
@kyeah I think this is part of the home page updates